specification/apicenter/resource-manager/Microsoft.ApiCenter/preview/2024-03-15-preview/apicenter.json

```diff
@@ -0,0 +1,4985 @@
+        ]
+      },
+      "readOnly": true
+    },
+    "Service": {
+      "type": "object",
+      "description": "The service entity.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServiceProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "description": "The managed service identities assigned to this resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ServiceListResult": {
+      "type": "object",
+      "description": "The response of a Service list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The Service items on this page",
+          "items": {
+            "$ref": "#/definitions/Service"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "ServiceProperties": {
+      "type": "object",
+      "description": "The properties of the service.",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "Provisioning state of the service.",
+          "readOnly": true
+        },
+        "restore": {
+          "type": "boolean",
+          "description": "Flag used to restore soft-deleted API Center service. If specified and set to 'true' all other properties will be ignored.",
+          "default": false
+        }
+      }
+    },
+    "ServiceUpdate": {
+      "type": "object",
+      "description": "The type used for update operations of the Service.",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "description": "The managed service identities assigned to this resource."
+        },
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/ServiceUpdateProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ServiceUpdateProperties": {
+      "type": "object",
+      "description": "The updatable properties of the Service.",
+      "properties": {
+        "restore": {
```

Comment:
consider using enum instead of boolean:
https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md

Reply:
same answer as above ^